LibraBFT/ImplShared/Util/Dijkstra/Error/Syntax.agda

```diff
@@ -0,0 +1,46 @@
+{- Byzantine Fault Tolerant Consensus Verification in Agda, version 0.9.
```

Comment:
To clarify - the outer module should be `EitherD` and the `Error.agda` file should be `EitherD.agda`

Reply:
Done!